bumps/webview/client/src/components/CorrelationViewPlotly.vue

```
@@ -70,7 +70,7 @@ async function fetch_and_draw(latest_timestamp?: string) {
         return;
       }
       // we are putting an array of numbers in customdata on the server side.
-      vars.value = ev.points[0].data.customdata as number[];
+      vars.value = ev.points[0]?.data.customdata as number[];
```

Again, I prefer to not hide errors. How do you reproduce the error that needs this change?